CompanyPaymentsProject/Program.cs

```diff
@@ -10,16 +10,23 @@ class Program
     {
         static void Main(string[] args)
         {
+            var company = new Company();
             
-            //na konsoli będzie można podać Id pracownika
-            //i wtedy w odpowiedzi suma wypłaconych pieniędzy
+            // możliwość dodawania pracowników
+            company.AddPerson(new Person {PersonID = 1, Name = "Jan", Surname = "Kowalski", Post.Assistant });
+            company.AddPerson(new Person {PersonID = 2, Name = "Anna", Surname = "Nowak", Post.Junior,});
+            company.AddPerson(new Person { PersonID = 3, Name = "John", Surname = "Doe", Post.Manager });
             
-            Console.WriteLine("Please enter the worker ID number to find out the worker salary.");
-            int userID = int.Parse(Console.ReadLine());
+            // dodanie wypłat dla pracowników
+            company.AddPayCheck(1);
```

Comment:
Tutaj jak dodasz datę do funkcji AddPaycheck to będziesz sobie musiała ją tutaj wygenerować, np. new DateTime(2020, 1, 1) czy mniej więcej coś takiego.